config.json

```diff
@@ -1,58 +1,1349 @@
+            "player": 1,
+            "player_threat_distance_factor": 2.0,
+            "threat_distance_factor": 1.0,
+            "system_installation": 3,
+            "dynamic_throttle": {
+                "max_distance_factor": 4.0,
+                "min_distance_factor": 0.25,
+                "target_fps": 30.0
+            }
+        },
+        "tractor": {
+            "max_relative_speed": 150,
+            "percent_to_tractor": 0.1,
+            "scoop": false,
+            "scoop_alpha_multiplier": 2.5,
+            "scoop_angle": 0.5,
+            "scoop_fov": 0.5,
+            "scoop_long_slices": 10,
+            "scoop_rad_slices": 10,
+            "cargo": {
+                "distance": 0.0,
+                "distance_own_rsize": 1.5,
+                "distance_tgt_rsize": 1.1,
+                "force_push": true,
+                "force_in": true
+            },
+            "disabled": {
+                "distance": 0.0,
+                "distance_own_rsize": 1.5,
+                "distance_tgt_rsize": 1.1,
+                "force_push": true,
+                "force_in": true
+            },
+            "faction": {
+                "distance": 0.0,
+                "distance_own_rsize": 2.2,
+                "distance_tgt_rsize": 2.2,
+                "force_push": true,
+                "force_in": true
+            },
+            "upgrade": {
+                "distance": 0.0,
+                "distance_own_rsize": 1.5,
+                "distance_tgt_rsize": 1.1,
+                "force_push": true,
+                "force_in": true
+            },
+            "others": {
+                "distance": 0.0,
+                "distance_own_rsize": 1.1,
+                "distance_tgt_rsize": 1.1,
+                "force_push": false,
+                "force_in": false
+            }
+        }
+    },
+
+    "player": {
+        "callsign": "",
+        "password": ""
+    },
+
+    "splash": {
+        "auto_hide": true
+    },
+
+    "terrain": {
+        "mass": 100.0,
+        "radius": 10000.0,
+        "xscale": 1.0,
+        "yscale": 1.0,
+        "zscale": 1.0
+    },
+
+    "test": {
+        "autodocker": false
+    },
 
-"constants": {
-    "megajoules_multiplier": 100,
-    "kilo": 1000,
-    "kj_per_unit_damage": 5400
-},
+    "unit": {
+        "default_aggressivity": 2.01
+    },
 
-"game_start": {
-    "default_mission": "main_menu.mission",
-    "introduction": 
-        "Welcome to Vega Strike!\nUse #8080FFTab#000000 to afterburn (#8080FF+,-#000000 cruise control),\n#8080FFarrows#000000 to steer.\nThe #8080FFt#000000 key targets objects; #8080FFspace#000000 fires at them & #8080FFa#000000 activates the SPEC drive.\nTo go to another star system, buy a jump drive for about 10000 credits,\nfly to a wireframe jump-point and press #8080FFj#000000 to warp to a near star.\nTarget a base or planet;\nWhen you get close a green box will appear. Inside the box, #8080FFd#000000 will land."    
-},
+    "warp": {
```

Comment:
change warp to ftl for consistency and neutrality.

Reply:
OK, I can do that.

Reply:
I think I might do it in a follow-up PR though, since the game engine code is already expecting "warp".